core/src/test/java/org/opensearch/sql/planner/physical/HighlightOperatorTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.planner.physical;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.opensearch.sql.data.type.ExprCoreType.STRING;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.opensearch.sql.data.model.ExprValue;
+import org.opensearch.sql.data.model.ExprValueUtils;
+import org.opensearch.sql.expression.DSL;
+
+public class HighlightOperatorTest extends PhysicalPlanTestBase {
+
+  @Test
+  public void highlight_all_test() {
```

Comment:
what is this testing?  that highlight all doesn't break anything?

Reply:
I suppose this should be renamed to `highlight_operator_test`. It is required for HighlightOperator coverage. The `*` isn't necessary for the highlight field.